bukuserver/response.py

```diff
@@ -1,17 +1,23 @@
 from enum import Enum
 from flask import jsonify
-from flask_api import status
+from flask_api.status import HTTP_200_OK, HTTP_400_BAD_REQUEST, HTTP_404_NOT_FOUND
 
 OK, FAIL = 0, 1
 
 
 class Response(Enum):
-    SUCCESS = (status.HTTP_200_OK, "Success.")
-    FAILURE = (status.HTTP_400_BAD_REQUEST, "Failure.")
-    INPUT_NOT_VALID = (status.HTTP_400_BAD_REQUEST, "Input data not valid.")
-    BOOKMARK_NOT_FOUND = (status.HTTP_404_NOT_FOUND, "Bookmark not found.")
-    TAG_NOT_FOUND = (status.HTTP_404_NOT_FOUND, "Tag not found.")
-    TAG_NOT_VALID = (status.HTTP_400_BAD_REQUEST, "Invalid tag.")
+    SUCCESS = (HTTP_200_OK, "Success.")
+    FAILURE = (HTTP_400_BAD_REQUEST, "Failure.")
+    INPUT_NOT_VALID = (HTTP_400_BAD_REQUEST, "Input data not valid.")
+    BOOKMARK_NOT_FOUND = (HTTP_404_NOT_FOUND, "Bookmark not found.")
+    TAG_NOT_FOUND = (HTTP_404_NOT_FOUND, "Tag not found.")
+    RANGE_NOT_VALID = (HTTP_400_BAD_REQUEST, "Range not valid.")
+    TAG_NOT_VALID = (HTTP_400_BAD_REQUEST, "Invalid tag.")
+
+    @staticmethod
+    def bad_request(message: str):
+        json = {status: Response.FAILURE.status, message: message}  # pylint: disable=E0602
```

Comment:
Er... maybe instead of disabling variable check, just use strings as dict keys? 😅